src/conn/conn_capacity.c

```diff
@@ -34,14 +34,26 @@ __capacity_config(WT_SESSION_IMPL *session, const char *cfg[])
     uint64_t chunkcache, total;
 
     conn = S2C(session);
-    chunkcache = total = 0;
+    cap = &conn->capacity;
 
+    chunkcache = total = 0;
     WT_RET(__wt_config_gets(session, cfg, "io_capacity.total", &cval));
-    if (cval.val != 0) {
-        if (cval.val < WT_THROTTLE_MIN)
-            WT_RET_MSG(session, EINVAL, "total I/O capacity value %" PRId64 " below minimum %d",
-              cval.val, WT_THROTTLE_MIN);
-        total = (uint64_t)cval.val;
+    if (cval.val != 0 && cval.val < WT_THROTTLE_MIN)
```

Comment:
Use idiom:
```
   if (cval.val != 0) {
        ...any other checks...
        cap->.... = (uint64_t)cval.val;
   }
```
as mentioned in previous comment.  It's not unsafe here, but it's good to always use a safe idiom.